app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,64 @@
+<h1>Task List</h1>
+
+<table>
+  <tr>
+    <th>
+      Name
+    </th>
+    <th>
+      Owner
+    </th>
+    <th>
+      Status
+    </th>
+    <th class="center">
+      Mark Done
+    </th>
+    <th class="center">
+      Details
+    </th>
+    <th class="center">
+      Delete
+    </th>
+  </tr>
+  <tbody>
+    <% @all_tasks.each do |task| %>
+      <% if task.completed_date.present? %>
+        <% @task_css = "completed" %>
```

Comment:
Generally, instance variables (`@something`) are used to pass data between the controller and the view. So it's probably better not to assign them values in the view (because some other view may be expecting the value from the controller).

In this case, what you are assigning is scoped within the `.each do` block, so you can just use a local variable, `task_css` (without the `@` prefix).